tests/test_cookiecutter_generation.py

```diff
@@ -362,6 +364,19 @@ def test_pycharm_docs_removed(cookies, context, use_pycharm, pycharm_docs_exist)
         assert has_pycharm_docs is pycharm_docs_exist
 
 
+@pytest.mark.parametrize("use_kedro", ["n", "y"])
+def test_kedro_in_pyproject(cookies, context, use_kedro):
+    context.update({"use_kedro": use_kedro})
+    result = cookies.bake(extra_context=context)
+    pyproject_contents = result.project.join("pyproject.toml").read()
+    kedro_package_check_list = ["kedro", "kedro-datasets", "kedro-airflow", "kedro-mlflow"]
+
+    if use_kedro == "y":
+        assert all([x in pyproject_contents for x in kedro_package_check_list]) == True
```

Comment:
Instead of if else, you can have the result as part of parametrize.

Reply:
I was using if/else because of the any and all functions being used depending on the value of use_kedro.  I am open to alternatives though!  What did you have in mind?

Reply:
I will also look into the lock file as part of the cookiecutter, but it might need to be added by the end user in their repo because it would be generated post cookie bake?

Reply:
Something like:
`@pytest.mark.parametrize("use_kedro, result", [("n", False), ("y", True)])`

Then assert
`assert all([x in pyproject_contents for x in kedro_package_check_list]) == result`